.github/workflows/ci_workflow.yml

```diff
@@ -225,6 +226,22 @@ jobs:
                 --config ${{ matrix.build-type }} \
                 -- -j4
         working-directory: _build
+      - name: Examples
+        run: |
+          # Build and run the examples as a standalone program linking 
+          # against the OpenEXR isntallation.
+          cmake -B . \
+                -S ../src/examples \
+                -DOpenEXR_DIR=../../_install/lib64/cmake/OpenEXR \
+                -DImath_DIR=../../_install/lib64/cmake/Imath \
+                -DCMAKE_BUILD_TYPE=${{ matrix.build-type }} \
+                -DCMAKE_CXX_STANDARD=${{ matrix.cxx-standard }} \
+                -DCMAKE_CXX_FLAGS=${{ matrix.cxx-flags }}
+          cmake --build . \
+                --config ${{ matrix.build-type }} \
+                -- -j4
```

Comment:
Not necessary; modern cmake with --build knows to use all the cores available.

Reply:
Just to clarify, are you referring to just the "-j4" part, or is there more than that?

Reply:
-j4 is not necessary